lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_non_nat_dhcp.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Does this test need to be added to `lte/gateway/python/integ_tests/defs.mk`?

Reply:
Probably yes. Which category would make the most sense?

Reply:
if it's sufficiently fast I would add it as a precommit test.